sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -79,12 +79,27 @@ export enum StudentProfileFormModes {
 
 export type StudentProfileFormModel = Pick<
   StudentProfileAPIOutDTO,
-  "firstName" | "lastName" | "gender" | "email" | "disabilityStatus"
+  "firstName" | "lastName" | "gender" | "email"
```

Comment:
The removed `disabilityStatus`(boolean) was also a form key in `studentprofile.json` with some logic around it (`"when": "disabilityStatus"`). Should it be removed or adjusted?

Reply:
the key is not removed, it's data type is modified.

![image](https://github.com/bcgov/SIMS/assets/54600590/9b7e3e27-3a4c-4339-a890-afd13ce9aede)